lib/handlers/payment_result_handler.dart

```diff
@@ -44,24 +53,85 @@ class PaymentResultHandler extends Handler {
       return;
     }
 
-    if (paymentResult.paymentInfo != null) {
+    final paymentInfo = paymentResult.paymentInfo;
+    final error = paymentResult.error;
+    if (paymentInfo != null) {
       final texts = context.texts();
       showFlushbar(
         context,
         messageWidget: SingleChildScrollView(
           child: Text(
-            paymentResult.paymentInfo!.paymentType == PaymentType.Received
+            paymentInfo.paymentType == PaymentType.Received
                 ? texts.successful_payment_received
                 : texts.home_payment_sent,
           ),
         ),
       );
-    } else if (paymentResult.error != null) {
-      _log.info("paymentResult error: ${paymentResult.error}");
+    } else if (error != null) {
+      _log.info("paymentResult error: $error");
+      if (error.ignoreGlobalFeedback) {
+        return;
+      }
       showFlushbar(
         context,
-        message: paymentResult.errorMessage(),
+        message: error.message,
       );
+      if (!error.validationError) {
```

Comment:
Which errors do we want to ignore here?

Reply:
Another piece of logic I copied from breez mobile https://github.com/breez/c-breez/issues/730#issuecomment-1892277574

We are ignoring "validation error", where validation error is any error where the message contains `rpc error` or the trace report is null or empty.

I don't have the context on breez mobile when that was made, I just kept it as it is working there; if you think it is not necessary on c-breez I can remove that.

Reply:
I think you won't get these strings in c-breez. Breez mobile works with LND in a different API and model.
Let's remove it.

Reply:
I see; I'll remove it.